code/script.js

```diff
@@ -1,9 +1,14 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
+// Find all elements with the class "accordion-item-header" and assign them to the variable "accordionItemHeader".
+const accordionItemHeaders = document.querySelectorAll(".accordion-item-header");
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+// Put all accordionItemHeader in the accordionItemHeaders ListeningStateChangedEvent, and toggle the class active depending on whether they already are active or not 
+accordionItemHeaders.forEach(accordionItemHeader => {
+  accordionItemHeader.addEventListener("click", event => {
+    accordionItemHeader.classList.toggle("active");
```

Comment:
Nice, happy to see other approach of adding some functionality than the one I introduced you to!